packages/volto/src/components/manage/Widgets/__snapshots__/TokenWidget.test.jsx.snap

```diff
@@ -64,6 +64,7 @@ exports[`renders a token widget component 1`] = `
                 >
                   <input
                     aria-autocomplete="list"
+                    aria-label="My field"
                     aria-labelledby="fieldset-default-field-label-my-field"
```

Comment:
Is it okay to have both aria-label and aria-labelledby?

Reply:
No, you are right. The problem was that this file is already fixed, so the modification I made now was not necessary. I will close this one and leave only v17 open. I will replicate the modification made to this file for the v17 one, to be compliant.

Reply:
@Wagner3UB Ah okay!